documentation/docs/getting-started/providers.md

```diff
@@ -55,139 +55,185 @@ goose also supports special "pass-through" providers that work with existing CLI
-
-    :::info Free Credits Offer
-    You'll receive $10 in free credits the first time you automatically authenticate with Tetrate through goose. This offer is available to both new and existing Tetrate users.
-    :::
-    1. Choose `Automatic setup with Tetrate Agent Router`. 
-    2. goose will open a browser window for you to authenticate with Tetrate, or create a new account if you don't have one already.
-    3. When you return to the goose desktop app, you're ready to begin your first session.
-    </TabItem>
-
-    <TabItem value="openrouter" label="OpenRouter">
-    1. Choose `Automatic setup with OpenRouter`. 
-    2. goose will open a browser window for you to authenticate with OpenRouter, or create a new account if you don't have one already.
-    3. When you return to the goose desktop app, you're ready to begin your first session.
-    </TabItem>
-
-    <TabItem value="others" label="Other Providers">
-    1. If you have a specific provider you want to use with goose, and an API key from that provider, choose `Other Providers`. 
-    2. Find the provider of your choice and click its `Configure` button. If you don't see your provider in the list, click `Add Custom Provider` at the bottom of the window. 
-    3. Depending on your provider, you'll need to input your API Key, API Host, or other optional [parameters](#available-providers). Click the `Submit` button to authenticate and begin your first session.
-    </TabItem>
-  </Tabs>
-  **To update your LLM provider and API key:** 
-  1. Click the <PanelLeft className="inline" size={16} /> button in the top-left to open the sidebar
-  2. Click the `Settings` button on the sidebar
-  3. Click the `Models` tab
-  4. Click `Configure providers`
-  5. Click your provider in the list
-  6. Add your API key and other required configurations, then click `Submit`
-
-  **To change your current model:**
-  1. Click the <PanelLeft className="inline" size={16} /> button in the top-left to open the sidebar
-  2. Click the `Settings` button on the sidebar
-  3. Click the `Models` tab
-  4. Click `Switch models`
-  5. Choose from your configured providers in the dropdown, or select `Use other provider` to configure a new one
-  6. Select a model from the available options, or choose `Use custom model` to enter a specific model name
-  7. Click `Select model` to confirm your choice
-
-  :::tip Shortcut
-  For faster access, click your current model name at the bottom of the app and choose `Change Model`.
-  :::
-
-  **To start over with provider and model configuration:**
-  1. Click the <PanelLeft className="inline" size={16} /> button in the top-left to open the sidebar
-  2. Click the `Settings` button on the sidebar
-  3. Click the `Models` tab
-  4. Click `Reset Provider and Model` to clear your current settings and return to the welcome screen
+
+**To update your LLM provider and API key:**
```

Comment:
Thanks for this PR. Can I ask one request?

hmm instead of deleting some of this stuff like the first time users information..we need to add yours PLUS keep the first -time users, open router, and other provider information. We actually just added the first-time users section this week at the request of our manager.